kata-deploy/scripts/kata-deploy.sh

```diff
@@ -65,7 +65,11 @@ function configure_crio() {
   runtime_path = "/opt/kata/bin/kata-fc"
 EOT
 
-	sed -i 's|\(\[crio\.runtime\]\)|\1\nmanage_network_ns_lifecycle = true|' "$crio_conf_file"
+  if `grep -Fq 'manage_network_ns_lifecycle' $crio_conf_file`; then
```

Comment:
@ganeshmaharaj How does this look?
```
# Replace if exists, else insert
grep -Fq 'manage_network_ns_lifecycle =' $crio_conf_file \
&& sed -i '/manage_network_ns_lifecycle =/c manage_network_ns_lifecycle = true' $crio_conf_file \
|| sed -i '/[crio.runtime]/a manage_network_ns_lifecycle = true' $crio_conf_file
```

Reply:
@krsna1729 guess they both do the same thing. Are there advantages to using one over the other that you are alluding to that I am not seeing?

Reply:
@ganeshmaharaj purely looks  :) less escapes